examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/LoginFragment.kt

```diff
@@ -40,11 +40,25 @@ class LoginFragment : Fragment() {
             }
         }
 
+        binding.resetSdkButton.setOnClickListener {
+            resetSdk()
+            navigateToConfigureFragment()
+        }
+
         return binding.root
     }
 
+    private fun resetSdk() {
+        Purchases.sharedInstance.close()
```

Comment:
Honestly, I'm not sure if there is a better way to reset the SDK... This doesn't fully clear the purchases instance (for example, the `sharedInstance` and most of the internal classes remains), but seems like the SDK doesn't provide a different way to do this. Ideally, the whole `Purchases` instance would get released but I don't think there is a way to do that currently.

Reply:
Good to know! People are probably using this function to clear all instances